November release highlights update (#2073)
Azure/azure-sdk: releases/2020-11/index.md

## Release Highlights

### Identity

We're glad to announce a new major release of our Identity package. This release includes standardized Managed Identity Credential support across languages, through which we add support to Azure Arc and Azure Service Fabric. This release also includes improvements to Visual Studio Code Credential, Device Code Credential and Interactive Browser Credential.

### Service Bus

> The new version 7 of the Service Bus libraries for C#, Python, Java and JavaScript provide the ability to share in some of the cross-service improvements made to the Azure development experience, such as using the new Azure Identity library to share a single authentication between clients and a unified diagnostics pipeline offering a common view of the activities across each of the client libraries. These libraries come with a single top level client for send and receive operations in contrast with multiple entry points from before, and a new administration client to perform CRUD operations on service bus entities. These libraries also include many updates, improvements, and a great deal of restructuring when compared to the previous versions, refer to the migration guides to hop on to the latest versions.

Reviewer: NIT
```suggestion
The new version 7 of the Service Bus libraries for C#, Python, Java, and JavaScript provide the ability to share in some of the cross-service improvements made to the Azure development experience, such as using the new Azure Identity library to share a single authentication between clients and a unified diagnostics pipeline offering a common view of the activities across each of the client libraries. These libraries come with a single top-level client for send and receive operations in contrast with multiple entry points from before, and a new administration client to perform CRUD operations on service bus entities. This version also includes many updates, improvements, and a great deal of restructuring when compared to the previous versions, refer to the migration guides to hop on to the latest versions.
```

> The new version 7 of the Service Bus libraries for C#, Python, Java and JavaScript provide the ability to share in some of the cross-service improvements made to the Azure development experience, such as using the new Azure Identity library to share a single authentication between clients and a unified diagnostics pipeline offering a common view of the activities across each of the client libraries. These libraries come with a single top level client called `ServiceBusClient` in contrast with multiple entry points from before, and a new `ServiceBusAdministrationClient` to perform CRUD operations on service bus entities. These libraries also include many updates, improvements, and a great deal of restructuring when compared to the previous versions, refer to the migration guides to hop on to the latest versions.

Reviewer 2: In Java, we dont have the ServicebusClient or the ServiceBusAdmininstrationClient, because we have builders there. So, I would recommend not using class names here

> These libraries come with a single top level client for send and receive operations in contrast with multiple entry points from before, and a new administration client to perform CRUD operations on service bus entities.

Author: sounds good! Thank you!

### Form Recognizer

We're releasing a new beta version of Form Recognizer. This version includes support for two new prebuilt recognition models for invoices and business cards, support for selection marks as a new fundamental form element, support for the bitmap image format in prebuilt model recognition and content recognition, language and locale arguments added to process document in different languages, and other additional properties added to response models.

### Text Analytics

> We're releasing a new beta version of Text Analytics. This version includes support for batch task processing and for the recognition of  various healthcare entities such as medication name, dosage, and frequency.

Reviewer 3: ```suggestion
We're releasing a new beta version of Text Analytics. This version includes support for batch task processing and for the recognition of various healthcare entities such as medication name, dosage, and frequency.
```

## Release Highlights

### Identity

We're glad to announce a new major release of our Identity package. This release includes standardized Managed Identity Credential support across languages, through which we add support to Azure Arc and Azure Service Fabric. This release also includes improvements to Visual Studio Code Credential, Device Code Credential and Interactive Browser Credential.

### Service Bus

Version 7 of the Azure Service Bus SDKs have now been released in all the four languages

> [C# - Nuget](https://www.nuget.org/packages/Azure.Messaging.ServiceBus/7.0.0), [Python - pypi](https://pypi.org/project/azure-servicebus/7.0.0/), [Java](wait for the release), [JS/TS - npm](https://www.npmjs.com/package/@azure/service-bus/v/7.0.0).

Reviewer: `[Java](wait for the release)`
Java isn't released yet, make sure to update the link before merging.

The latest set of packages are built using the new [Azure SDK guidelines](https://github.com/Azure/azure-sdk/blob/master/README.md) which gives the consistency across all Azure SDK client libraries. These libraries enable users leverage the Azure Service Bus messaging service in their applications and is our effort to make integrating Service Bus quick and painless, allowing you to quickly jumpstart your applications and get running.

> The new version 7 of the Service Bus libraries provide the ability to share in some of the cross-service improvements made to the Azure development experience, such as using the new `@azure/identity` library to share a single authentication between clients and a unified diagnostics pipeline offering a common view of the activities across each of the client libraries. These libraries come with a single top level client called `ServiceBusClient` in contrast with multiple entry points from before, and a new `ServiceBusAdministrationClient` to perform CRUD operations on service bus entities. These libraries also include many updates, improvements and a great deal of restructuring when compared to the previous versions, refer to the migration guides to hop on to the latest versions and follow the links below for more info.

Reviewer: ```suggestion
The new version 7 of the Service Bus libraries provide the ability to share in some of the cross-service improvements made to the Azure development experience, such as using the new `@azure/identity` library to share a single authentication between clients and a unified diagnostics pipeline offering a common view of the activities across each of the client libraries. These libraries come with a single top level client called `ServiceBusClient` in contrast with multiple entry points from before, and a new `ServiceBusAdministrationClient` to perform CRUD operations on service bus entities. These libraries also include many updates, improvements, and a great deal of restructuring when compared to the previous versions, refer to the migration guides to hop on to the latest versions.
```

> The new version 7 of the Service Bus libraries provide the ability to share in some of the cross-service improvements made to the Azure development experience, such as using the new `@azure/identity` library to share a single authentication between clients and a unified diagnostics pipeline offering a common view of the activities across each of the client libraries. These libraries come with a single top level client called `ServiceBusClient` in contrast with multiple entry points from before, and a new `ServiceBusAdministrationClient` to perform CRUD operations on service bus entities. These libraries also include many updates, improvements and a great deal of restructuring when compared to the previous versions, refer to the migration guides to hop on to the latest versions and follow the links below for more info.

Reviewer 4: Love all the detail but I think you can cut this second paragraph as it is a bit lengthy.

Reviewer 2: @czubair Instead, can we simply link to the blog that @richardpark-msft is working on? I see that in the last month's highlights, we did something similar for metrics advisor: https://github.com/Azure/azure-sdk/blob/9fc303091a697aeab8b183ea094936b7cbf5b744/releases/2020-10/index.md

Author: I've reduced this to this one paragraph: https://github.com/Azure/azure-sdk/pull/2073/files#diff-72fedcd9a0c5d8a64d2b941d1242f8d5b580e3ae2965b6b9d685b306c8569927R21

Reviewer 4: @ramya-rao-a once Richard has published the blog then yes, we can update this to link to his post.

> The new version 7 of the Service Bus libraries provide the ability to share in some of the cross-service improvements made to the Azure development experience, such as using the new `@azure/identity` library to share a single authentication between clients and a unified diagnostics pipeline offering a common view of the activities across each of the client libraries. These libraries come with a single top level client called `ServiceBusClient` in contrast with multiple entry points from before, and a new `ServiceBusAdministrationClient` to perform CRUD operations on service bus entities. These libraries also include many updates, improvements and a great deal of restructuring when compared to the previous versions, refer to the migration guides to hop on to the latest versions and follow the links below for more info.

Reviewer 5: ```suggestion
The new version 7 of the Service Bus libraries provide the ability to share in some of the cross-service improvements made to the Azure development experience, such as using the new Azure Identity library to share a single authentication between clients and a unified diagnostics pipeline offering a common view of the activities across each of the client libraries. These libraries come with a single top level client called `ServiceBusClient` in contrast with multiple entry points from before, and a new `ServiceBusAdministrationClient` to perform CRUD operations on service bus entities. These libraries also include many updates, improvements and a great deal of restructuring when compared to the previous versions, refer to the migration guides to hop on to the latest versions and follow the links below for more info.
```

Dropping `@azure/identity` as it's very specific to the JS/TS SDK.

Author: Thank you! I've made the update. Here is how it looks now: https://github.com/Azure/azure-sdk/pull/2073/files#diff-72fedcd9a0c5d8a64d2b941d1242f8d5b580e3ae2965b6b9d685b306c8569927R21

### Form Recognizer

We're releasing a new beta version of Form Recognizer. This version includes support for two new prebuilt recognition models for invoices and business cards, support for selection marks as a new fundamental form element, support for the bitmap image format in prebuilt model recognition and content recognition, language and locale arguments added to process document in different languages, and other additional properties added to response models.

> 

Reviewer 3: Let me sneak in one for text analytics too:

```suggestion

### Text Analytics

We're releasing a new beta version of Text Analytics. This version includes support for batch task processing and for the recognition of  various healthcare entities such as medication name, dosage, and frequency.
```
